ens-names/README.md

```diff
@@ -0,0 +1,3 @@
+# ENS Names
+
+NOTE: At the time of writing, the ens substream only works for ENS registered with ETH Base Registrar contract.
```

Comment:
you should add a more detailed description about what this substream does

Reply:
Sure